index.html

```diff
@@ -87,8 +88,13 @@
 			<label for="my_house">I Did ( My spouse gets nothing! )</label>
 
 			<input class="input" type="radio" name="whos_property" value="spouses_house" id="spouses_house" />
-			<label for="spouces_house">My Spouse Did ( I want all I can get! )</label>
+			<label for="spouses_house">My Spouse Did ( I want all I can get! )</label>
+		</div>
+	</div>
 
+	<div class="row">
+		<div class="col-md-offset-4 col-md-4 text-center">
+			<button class="btn btn-primary submit">Submit</button>
```

Comment:
What you had was correct and would work perfect except for the way that the javascript is written it didn't work quite right (as we discussed last night).
All I did here was change so that the "click" event triggers the call to processChanges. Take a look at the "$(button.submit)" code in the javascript.